# STRique

*STRique is no longer maintained!   
It may still work in py3.6 and py3.7 virtual environments as long as the ONT fast5 format is not changed.*

[![Latest GitHub release](https://img.shields.io/github/release-pre/giesselmann/STRique.svg)](https://github.com/giesselmann/STRique/releases/latest)
[![Build Status](https://www.travis-ci.com/giesselmann/STRique.svg?branch=master)](https://www.travis-ci.com/giesselmann/STRique)
[![Docker Cloud Build Status](https://img.shields.io/docker/cloud/build/giesselmann/strique.svg)](https://hub.docker.com/r/giesselmann/strique/)
[![Documentation Status](https://readthedocs.org/projects/strique/badge/?version=latest)](https://strique.readthedocs.io/en/latest/?badge=latest)

<img align="right" src="https://github.com/giesselmann/strique/blob/master/docs/images/workflow.png" width="30%" hspace="20">

STRique is a python package to analyze repeat expansion and methylation states of short tandem repeats (STR) in Oxford Nanopore Technology (ONT) long read sequencing data.

Please refer to the **[documentation](https://strique.readthedocs.io/en/latest/)** for installation and usage instructions.

To get started the **[installation](https://strique.readthedocs.io/en/latest/installation/prerequisites/)** chapter describes the available installation options depending on the operation system and already existing environments.

In the **[Usage](https://strique.readthedocs.io/en/latest/examples/intro/)** chapter the stepwise analysis of nanopore reads covering repeat expansions is demonstrated.

## Citation

If you use STRique in your work please cite:

> Giesselmann, P., Br√§ndl, B., Raimondeau, E. et al. Analysis of short tandem repeat expansions and their methylation state with nanopore sequencing. Nat Biotechnol (2019) doi:10.1038/s41587-019-0293-x
